### PR TITLE
Bug 1960758: test/extended/cli/mustgather: Local-only getPluginOutputDir

### PR DESCRIPTION
**Pull request by wking**
Removing the oc image calculations which we grew in 17b5a92349
(support old and new must-gather output, 2019-09-04, #23724).  This
pivot avoids failing on:

```go
imageId, ok := imageutil.ResolveLatestTaggedImage(stream, "latest")
o.Expect(ok).To(o.BeTrue())
```

like [this job][1] did.  Checking the contents of that job's must-gathers:

```console
$ curl -s https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_oc/833/pull-ci-openshift-oc-master-e2e-metal-ipi-ovn-ipv6/1396775908324413440/artifacts/e2e-metal-ipi-ovn-ipv6/gather-must-gather/artifacts/must-gather.tar | tar -tvz | head -n5
drwx------ 1001210000/root   0 2021-05-24 05:38 .
-rw------- 1001210000/root 3406540 2021-05-24 05:38 event-filter.html
-rw------- 1001210000/root  758413 2021-05-24 05:38 must-gather.log
drwxr-xr-x 1001210000/root       0 2021-05-24 05:38 registry-build02-ci-openshift-org-ci-op-s1s14k72-stable-sha256-f43d80e9d24970297d32fc572b7fdd8349e1700f64785c0ed3b7f75c15bc7750
drwxr-xr-x 1001210000/root       0 2021-05-24 05:38 registry-build02-ci-openshift-org-ci-op-s1s14k72-stable-sha256-f43d80e9d24970297d32fc572b7fdd8349e1700f64785c0ed3b7f75c15bc7750/cluster-scoped-resources
```

and:

```console
$ curl -s https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_oc/833/pull-ci-openshift-oc-master-e2e-metal-ipi-ovn-ipv6/1396775908324413440/artifacts/e2e-metal-ipi-ovn-ipv6/gather-audit-logs/artifacts/audit-logs.tar | tar -tvz | head -n4
drwx------ 1001210000/root   0 2021-05-24 05:41 .
-rw------- 1001210000/root 3336 2021-05-24 05:41 event-filter.html
drwxr-xr-x 1001210000/root    0 2021-05-24 05:41 registry-build02-ci-openshift-org-ci-op-s1s14k72-stable-sha256-f43d80e9d24970297d32fc572b7fdd8349e1700f64785c0ed3b7f75c15bc7750
drwxr-xr-x 1001210000/root    0 2021-05-24 05:41 registry-build02-ci-openshift-org-ci-op-s1s14k72-stable-sha256-f43d80e9d24970297d32fc572b7fdd8349e1700f64785c0ed3b7f75c15bc7750/audit_logs
```

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_oc/833/pull-ci-openshift-oc-master-e2e-metal-ipi-ovn-ipv6/1396775908324413440

### CODE REVIEW

**Comment by wking** [comment]
[`should not have pod creation failures during install`][1], [`TestTriggers_manual` and `Managed cluster should set requests but not limits`][2], and [`volume snapshot controller should check snapshot fields...`][3] are all orthogonal, so I think we're good to go here.

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/26185/pull-ci-openshift-origin-master-e2e-gcp/1397721071779581952
[2]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/26185/pull-ci-openshift-origin-master-e2e-metal-ipi-ovn-ipv6/1397721071909605376
[3]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/26185/pull-ci-openshift-origin-master-e2e-aws-csi/1397721071641169920

File: test/extended/cli/mustgather.go [hunk 1]
```diff
@@ -348,16 +345,29 @@ var _ = g.Describe("[sig-cli] oc adm must-gather", func() {
...
-	imageId, ok := imageutil.ResolveLatestTaggedImage(stream, "latest")
-	o.Expect(ok).To(o.BeTrue())
-	pluginOutputDir := path.Join(tempDir, regexp.MustCompile("[^A-Za-z0-9]+").ReplaceAllString(imageId, "-"))
-	fileInfo, err := os.Stat(pluginOutputDir)
-	if err != nil || !fileInfo.IsDir() {
-		pluginOutputDir = tempDir
+	dir := ""
+	for _, file := range files {
+		if file.IsDir() {
+			if dir != "" {
```
Comment (soltysh) [inline test/extended/cli/mustgather.go:360]: Not sure we need this kind of check, but it sure doesn't hurt :wink:

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

File: test/extended/cli/mustgather.go (hunk 1 above)
Comment (wking) [reply test/extended/cli/mustgather.go:360]: This is the "I'm an old-style must gather" check.  I'm not sure how many tests we still run with `oc` that old, but the outgoing `if err != nil || !fileInfo.IsDir() {` was also trying to figure out old vs. new, so keeping this check just rolls that coverage along.

**Comment by wking** [comment]
Neither of the two failing test-cases in [the e2e-metal-ipi-ovn-ipv6job][1] are related.  Can we `/override ci/prow/e2e-metal-ipi-ovn-ipv6`?

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/26185/pull-ci-openshift-origin-master-e2e-metal-ipi-ovn-ipv6/1398243679474290688

**Pull request merged**